src/blueprints/tv2/helpers/tv2-show-style-blueprint-configuration-mapper.ts

```diff
@@ -23,6 +24,7 @@ interface CoreShowStyleBlueprintConfiguration {
   BreakerConfig: CoreBreaker[]
   Transitions: Transition[]
   ShowstyleTransition: string
+  LYDConfig: CoreAudioBedConfiguration[]
```

Comment:
is this some danish sneaking in? 😄 or is it an actual abbreviation?

Reply:
It is danish sneaking in from the old code base. So we will have to live with this until we control ingest.